src/contributing.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: layouts/base.njk
+---
+
+Welcome! First off; _**THANK YOU**_ for your interest in contributing to Zinc!
+Your time is valuable, so let's get started!
+
+## Ground Rules
+
+Zinc moves forward thanks to two kinds of workers. _Contributors_, who suggest
+and build changes to our software; and _Maintainers_, who decide if, when, and
+how to apply the suggestions. Anyone and everyone can be a Contributor! All we
+ask is that Contributors follow a few [Ground Rules][contributor-ground-rules]
+to help Zinc Maintainers and Contributors work well together.
+
+The keywords _must_, _must not_, and _should_ in this document are to be
+interpreted as described in [RFC 2119][rfc-2119].
+
+### Contributor Ground Rules
+
+- Contributors _must_ comply with Zinc's [code of conduct][code-of-conduct].
```

Comment:
Link is incorrect, or maybe we haven't actually created this content?

Reply:
Yea, I was... hoping to have energy to move the one in compensated but don't have the spoons yet :(.

Reply:
Link is fixed!